src/config.ts

```diff
@@ -1,14 +1,61 @@
 const {
-    NATS_URL = "nats://localhost:4222",
+    AUTH_SERVER = "default",
+    AUTH_PROTOCOL = "",
+    AUTH_URL = "",
+    SECURE = true,
+    NATS_URL = "nats://nats:4222",
     HTTP_PORT = 8000,
-    DOCS_GATHERING_TIMEOUT = 50
+	DOCS_GATHERING_TIMEOUT = 50
 } = process.env;
 
 const NUM_HTTP_PORT = Number(HTTP_PORT);
 const NUM_DOCS_GATHERING_TIMEOUT = Number(DOCS_GATHERING_TIMEOUT);
+const BOOL_SECURE = Boolean(SECURE);
+
+if (!AUTH_SERVER) {
+    throw new Error(
+        `Please provide a valid authorization server via AUTH_SERVER. For instance, use default`
+    )
+}
+
+if (!AUTH_PROTOCOL) {
+    throw new Error(
+        `Please provide a valid protocol to use via AUTH_PROTOCOL. For instance, use https`
+    )
+}
+
+if (!AUTH_URL) {
+    throw new Error(
+        `Please provide a valid authorization server url to use via AUTH_URL. For instance, localhost:8001`
+    )
+}
+
+if (!SECURE) {
+    throw new Error(
+        `Please provide a valid security configuration via SECURE.`
+    )
+} else {
+    console.error("/!\\ AIRLOCK IS RUNNING IN INSECURE MODE, PLEASE USE THIS CONFIGURATION ONLY FOR DEVELOPMENT PURPOSES /!\\")
```

Comment:
this should be a warning instead of an error I think. if it was an error the app should crash, but in our case we keep it running, we just warn that it's not in a good state